src/introduction/get.md

```diff
@@ -8,8 +8,11 @@ the University of Cambridge includes a docker environment and pre-built
 compilers and emulators for both CHERI-RISC-V and Morello<!-- XXX as well as
 [pre-built exercise binaries](./run-release.md) that can be run out of the box
 without first needing to cross-compile-->.  Follow the instructions in the
-release notes and then proceed to [check out the exercises](./get-us.md) and
-then to [cross compilation](./cross-ccc-release.md).
+[release notes](https://cheri-dist.cl.cam.ac.uk/) and make sure you unpack
```

Comment:
I waffled about making "release notes" a link.  Maybe for ASPLOS'22 specifically we should be linking to what we know will be the current release notes (i.e., https://cheri-dist.cl.cam.ac.uk/releases/2021.08/relnotes.html) and in the future just link to the distribution root as you have here?

Reply:
I agree.